src/resources/css/app.css

```
@@ -1,4 +1,117 @@
+.content-header {
+  position: relative;
+  padding: 30px 30px 0px;
+}
+
+
+
+.main-footer {
+  position: relative;
+  background-color: var(--dark-body);
+  color: var(--white);
+  border-color: rgba(--white, 0.12);
```

Biome (2.1.2)

[error] CSS variables '--white' is used without the 'var()' function

CSS variables should be used with the 'var()' function to ensure proper fallback behavior and browser compatibility.

(lint/correctness/noMissingVarFunction)